Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py

```diff
@@ -1371,25 +1372,44 @@ def get_sa_activities_command(account_id, client_id, auth_token, hostname, path,
         return_error(message='Please enter hostname or path', error='')
 
 
-def send_usage_data(account_id, email, company_name, USE_SSL):
-    url = 'https://bojxxu8tfa.execute-api.eu-west-1.amazonaws.com/v1/integrations'
+def send_usage_data(auth_token, account_id, client_id, msg_type, USE_SSL):
+    url = "https://api-msp-telemetry.malwarebytes.com/data"
+
+    headers = {'Content-Type': "application/json"}
+
+    tz = pytz.timezone('Europe/Moscow')
+    dt = datetime.now()
+    loc_dt = tz.localize(dt).replace(microsecond=0)
+    v = loc_dt.isoformat()
+    temp = v.split('+')
+    timestamp = temp[0] + 'Z'
+
+    body = {
+        "timestamp": timestamp,
+        "integration_code": "TA-CX",
+        "integration_name": "Cortex XSOAR",
+        "integration_app": "Malwarebytes",
+        "integration_app_version": "1.1.6",
+        "nebula_account_id": account_id,
+        "ov_account_id": "",
+        "mbbr_license_key": "",
+        "api_client_id": client_id,
+        "custom_fields": [],
+        "msg_type": msg_type,
+        "token": auth_token
+    }
 
-    if company_name and email:
-        data = {
-            "nebula_account_id": account_id,
-            "integration_name": "Cortex XSOAR",
-            "customer_name": company_name,
-            "email": email
-        }
-    else:
-        data = {
-            "nebula_account_id": account_id,
-            "integration_name": "Cortex XSOAR",
-            "customer_name": "N/A",
-            "email": "N/A"
-        }
     try:
-        requests.post(url, data=json.dumps(data), headers=None, verify=USE_SSL)
+        response = requests.request("POST", url, data=json.dumps(body), headers=headers, verify=USE_SSL)
+
+        if response.status_code == 200:
+            data = json.loads(response.text)
+            if data.get("statusCode") == 201:
+                return True
+            else:
+                return False
+        else:
+            return False
```

Comment:
You are not using the returned booleans.
I think the previous message was better, or alternatively you should get the returned boolean in the main function and print the relevant result to a log.

Reply:
The previous message was producing an error in the test function, that's why I changed it. found that in the morning. Also, I don't think this function needs to be logged as this is only for collecting usage analytics for us, it's not used by the customers in any way.

Reply:
Ok :) it's your choice.